stackstac/rio_reader.py

```diff
@@ -304,6 +304,7 @@ def __init__(
         dtype: np.dtype,
         fill_value: Union[int, float],
         rescale: bool,
```

Comment:
Also, the docstring for the `rescale` parameter should be updated after this to mention that the values come from STAC metadata.

Reply:
Updated